[Workers AI] Revise embedded function calling get started guide (#15389)
cloudflare/cloudflare-docs: content/workers-ai/function-calling/embedded/_index.md

Embedded function calling can be used to easily make complex agents that interact with websites and APIs, like using natural language to create meetings on Google Calendar, saving data to Notion, automatically routing requests to other APIs, saving data to an R2 bucket - or all of this at the same time. All you need is a prompt and an OpenAPI spec to get started.

{{<Aside type="warning" header="REST API support">}}

> Embedded function calling depends on features native to the Worker's platform. Hence, embedded function calling is only supported via Cloudflare Workers, not via the REST API.

Reviewer: ```suggestion
Embedded function calling depends on features native to the Workers platform. This means that embedded function calling is only supported via [Cloudflare Workers](/workers-ai/get-started/workers-wrangler/), not via the [REST API](/workers-ai/get-started/rest-api/).
```

{{</Aside>}}

## Get Started

### 1. Create a Worker project with Workers AI

>  Follow the general [Workers AI Get Started Guide](/workers-ai/get-started/workers-wrangler/) until step 2.

Reviewer: ```suggestion
 Follow the [Workers AI Get Started Guide](/workers-ai/get-started/workers-wrangler/) until step 2.
```

};
```

This example imports the utils with `import { createToolsFromOpenAPISpec, runWithTools, autoTrimTools } from "@cloudflare/ai-utils"` and follows the  API reference below. 

The `ai-utils package` is also open-sourced on [Github](https://github.com/cloudflare/ai-utils).

### 4. Local development & deployment

>  Follow the [Workers AI Get Started Guide'](/workers-ai/get-started/workers-wrangler/)s  step 4 and 5 for local development and deployment.

Reviewer: ```suggestion
 Follow steps 4 and 5 of the [Workers AI Get Started Guide](/workers-ai/get-started/workers-wrangler/) for local development and deployment.
```

{{<Aside type="note" header="Workers AI Embedded Function Calling charges">}}

> Embedded function calling runs Workers AI inference requests. Stanard charges for inference (e.g. tokens) usage will be charged.

Reviewer: ```suggestion
Embedded function calling runs Workers AI inference requests. Standard charges for inference (e.g. tokens) usage will be charged.
```
